src/main/java/com/cloudogu/smeagol/wiki/domain/CommitRepository.java

```diff
@@ -0,0 +1,5 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+public interface CommitRepository {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Annotate the "CommitRepository" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1609)